net-libs/nodejs/nodejs20-20.9.0.recipe

```diff
@@ -56,11 +62,12 @@ BUILD_PREREQUIRES="
 	cmd:ninja
 	cmd:python3
 	cmd:which
+	cmd:pkg_config
```

Comment:
Entries should be sorted alphabetically. Also, `cmd:pkg_config` will need to be changed to `cmd:pkg_config$secondaryArchSuffix` in order to work on 32 bit.  There are two pkg_config packages on 32 bit and nodejs20 needs to have the secondary one active.

Reply:
Done